roles/metrics/files/backup-influxdb.bash

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+remove_temporary_directory () {
+    if [[ -n $backup_dir ]]; then
```

Comment:
While this is reasonable thing to do and will most probably not fail since `backup_dir` is set just after setting the trap, this will make the script fail due to `set -o nounset`.

Reply:
Corrected